### DISCUSSION TITLE
Looking for a project

### DISCUSSION THREAD

**Discussion by Potatoasad** [williamgilpin/cphy#27] [projects]
Hi all,

I currently don't have any group members and am looking for a group to join. I'm a 5th year physics grad and I work on doing gravitational wave data analysis, black hole population statistics and black hole perturbation theory. 

Best,
Asad Hussain.
ah54594

**Comment by abao1999** [comment]
@sap4423 (Savannah) you may be interested?

**Comment by williamgilpin** [comment]
@Art-of-FUgue maybe of interest?

**Comment by Art-of-FUgue** [comment]
Hi:
As suggested by professor, I am interested in this topic. However, my background might not be very sufficient. Please let me know if you would like me to join. 
Thank you for your attention.

Best regards,
Hongming Zhang
hz6835
